.github/workflows/ci.yml

```diff
@@ -1,17 +1,13 @@
 name: "CI"
 
 on:
-  pull_request:
   push:
-    branches:
-      - "master"
```

Comment:
This was originally added in #856 without much discussion and while I agree that it may make sense in some projects, I don't think it's something that's very beneficial for Ratchet. In particular, outside contributors and maintainers alike would usually use a fork-based workflow that would use short-lived feature branches that do not usually match the given branch name. With this removed, GitHub Actions will run on any branch name both in this upstream repository and any forks. I think this is the better default here unless we have a clearer use case.